### PR TITLE
[icons] feat: new icons for low & high voltage power poles

### PR DESCRIPTION
**Pull request by RexTheInnkeeper**
#### Checklist

- [ ] Includes tests
- [ ] Update documentation

#### Changes proposed in this pull request:

added low and high voltage power pole icons

#### Reviewers should focus on:

make sure everything renders ok and correct files were changed

#### Screenshot

Poles 20px
Poles

### CODE REVIEW

**Comment by palantirtech** [comment]
Thanks for your interest in [palantir/blueprint](https://github.com/palantir/blueprint), @RexTheInnkeeper! Before we can accept your pull request, you need to sign our contributor license agreement - just visit https://cla.palantir.com/ and follow the instructions. Once you sign, I'll automatically update this pull request.

File: packages/icons/icons.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 [
-    {
+   {
```
Comment (adidahiya) [inline packages/icons/icons.json]:
```suggestion
    {
```

**Review by adidahiya — CHANGES_REQUESTED** [review]
these icons don't show up correctly in the [docs preview build](https://output.circle-artifacts.com/output/job/ad292ee8-d0f6-41bc-a746-270353974a59/artifacts/0/packages/docs-app/dist/index.html#icons):

image

image

**Comment by adidahiya** [comment]
@RexTheInnkeeper there are still some issues with the 20px high voltage icon:

image

File: resources/icons/20px/high-voltage-pole.svg [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<svg width="20" height="20" viewBox="0 0 20 20" fill="none" xmlns="http://www.w3.org/2000/svg">
+<g clip-path="url(#clip0_2240_7)">
```
Comment (adidahiya) [inline resources/icons/20px/high-voltage-pole.svg]: I think you need to remove this `<g>`

File: resources/icons/20px/high-voltage-pole.svg [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+<path fill-rule="evenodd" clip-rule="evenodd" d="M7 0C6.44772 0 6 0.447716 6 1V4H4V2C4.55228 2 5 1.55228 5 1C5 0.447715 4.55228 0 4 0H1C0.447715 0 0 0.447715 0 1C0 1.55228 0.447715 2 1 2H2V4H1C0.447715 4 0 4.44772 0 5C0 5.55228 0.447716 6 1 6H2V8H1C0.447715 8 0 8.44771 0 9C0 9.55228 0.447715 10 1 10H4C4.55228 10 5 9.55228 5 9C5 8.44771 4.55228 8 4 8V6H6V19C6 19.5523 6.44772 20 7 20C7.41998 20 7.77948 19.7411 7.92763 19.3742L9.99999 14.5387L12.0728 19.3753C12.2213 19.7416 12.5805 20 13 20C13.5523 20 14 19.5523 14 19V6H16V8C15.4477 8 15 8.44771 15 9C15 9.55228 15.4477 10 16 10H19C19.5523 10 20 9.55228 20 9C20 8.44771 19.5523 8 19 8H18V6H19C19.5523 6 20 5.55228 20 5C20 4.44772 19.5523 4 19 4H18V2H19C19.5523 2 20 1.55228 20 1C20 0.447715 19.5523 0 19 0H16C15.4477 0 15 0.447715 15 1C15 1.55228 15.4477 2 16 2V4H14V1C14 0.447715 13.5523 0 13 0H7ZM8 2V4H12V2H8ZM12 14.1282L11.088 12.0001L12 9.87201V14.1282ZM8.51647 6L9.99999 9.46153L11.4835 6H8.51647ZM8.91202 12.0001L8 9.87208V14.1282L8.91202 12.0001Z" fill="black"/>
+</g>
+<defs>
```
Comment (adidahiya) [inline resources/icons/20px/high-voltage-pole.svg]: also remove these unnecessary `<defs>`

**Review by adidahiya — CHANGES_REQUESTED** [review]

**Review by adidahiya — APPROVED** [review]
looks good now:

image

I'm not sure if we'll keep a distinct category for "Utilities" icons like this, but it's fine for now. I'll probably reorganize some icons into a category for "map markers" or something like that.

**Pull request merged**